emanote/src/Emanote/Route/Ext.hs

```diff
@@ -15,6 +15,7 @@ data SourceExt = SourceExt
 data FileType a where
   LMLType :: LML -> FileType SourceExt
   Yaml :: FileType SourceExt
+  Xml :: FileType SourceExt
```

Comment:
This should be `Xml :: FileType ()` since `.xml` is not a source file (it is rather an output file).